app/query_objects/filtered_hotels.rb

```diff
@@ -1,7 +1,9 @@
 class FilteredHotels
+  ALLOWED_PARAMS = %i[search stars min_rating max_rating near].freeze
+  DEFAULT_RADIUS = 100_000
```

Comment:
As far as I remember, radius should `100` if you want to search hotels around 100 km